.buildkite/pipeline.yml

```
@@ -262,6 +262,12 @@ steps:
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --idealized_insolation false --vert_diff true --surface_scheme monin_obukhov --moist equil --rad allskywithclear --precip_model 0M --job_id sphere_aquaplanet_rhoe_equilmoist_allsky --dt 400secs --t_end 5days --regression_test true"
         artifact_paths: "sphere_aquaplanet_rhoe_equilmoist_allsky/*"
 
+      - label: ":computer: held suarez (ρe_tot) equilmoist with EDMF"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff false --surface_scheme monin_obukhov --moist equil --anelastic_dycore false --forcing held_suarez --job_id sphere_held_suarez_rhoe_equilmoist_mo_edmf --regression_test true --dt 1secs --t_end 10mins --FLOAT_TYPE Float64 --turbconv edmf --turbconv_case Bomex --config sphere --microphy 0M --z_stretch false --dt_save_to_sol 5mins --dt_save_to_disk 5mins"
```

```suggestion
        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff false --moist equil --anelastic_dycore false --job_id sphere_baroclinic_wave_rhoe_equilmoist_mo_edmf --regression_test true --dt 1secs --t_end 10mins --FLOAT_TYPE Float64 --turbconv edmf --turbconv_case Bomex --config sphere --microphy 0M --z_stretch false --dt_save_to_sol 5mins --dt_save_to_disk 5mins"
```
Since you are only running it for 10 min, I think baroclinic wave is a better test case for whether it works as it starts with some moisture (the initial condition is dry of other cases). I also removed `--surface_scheme` because it doesn't matter when using `--vert_diff false`, as we discussed before.

---

And if this works, maybe we can add another job in ci with all the physical tendencies turned on.